ffs/integrationtest/integration_test.go

```diff
@@ -139,15 +147,15 @@ func TestInfo(t *testing.T) {
 	})
 
 	r := rand.New(rand.NewSource(22))
-	n := 3
+	n := 1
 	for i := 0; i < n; i++ {
 		cid, _ := addRandomFile(t, r, ipfs)
 		jid, err := fapi.PushConfig(cid)
 		require.Nil(t, err)
 		requireJobState(t, fapi, jid, ffs.Success)
 	}
 
-	t.Run("WithThreeAdds", func(t *testing.T) {
+	t.Run("WithOneAdd", func(t *testing.T) {
```

Comment:
Related with not being able to run more than one deal mentioned before and in PR description.